rpcs3/Emu/RSX/GL/OpenGL.h

```diff
@@ -6,7 +6,7 @@
 #ifdef _WIN32
 #include <Windows.h>
 #include "GL/gl.h"
-#include "GL/glext.h"
+#include "externals/GL/glext.h"
```

Comment:
(plus "externals" is likely to be removed by linux packager to use system wise includes)